.github/workflows/post_publish.yml

```
@@ -51,3 +51,52 @@ jobs:
     if: ${{ github.event.workflow_run.conclusion == 'failure' }}
     steps:
       - run: echo 'The triggering workflow failed'
+
+
+  registry-check:
+    # strategy:
+    #   matrix:
+    #     os: [ubuntu-latest, windows-latest, macOS-latest]
+    # runs-on: ${{ matrix.os }}
+    # runs-on: ubuntu-latest
+    runs:
+      using: "composite"
+      steps:
+        - using: composite
+        - name: Registry Version Check
+          id: registry_latest_ver
+          shell: bash
+          run: |
+            curl -s "https://registry.terraform.io/v2/providers/323/provider-versions/latest" | jq '.data.attributes.version' >> "$GITHUB_OUTPUT"
```

For the output to get registered properly, you'll need whatever's being sent to `$GITHUB_OUTPUT` to be in the format of `key=value`

```suggestion
            echo "current=$(curl -s https://registry.terraform.io/v2/providers/323/provider-versions/latest | jq '.data.attributes.version')" >> "$GITHUB_OUTPUT"
```